### PR TITLE
CMS-427: Build "Preview" page elements

### PR DESCRIPTION
**Pull request by duncan-oxd**
### Jira Ticket

CMS-427

### Description

The last page of elements/styles to build out (I think). The next step will be to set up the API and then turn these static HTML elements into React components when the API is ready.

### CODE REVIEW

**Review by ayumi-oxd — APPROVED** [review]
LGTM 👍 A button "Back to {PARK NAME} dates" is needed on the top of the preview page but we can add it later.

File: frontend/src/router/pages/ReviewChanges.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  // spacing between date range values
+  td:not(:last-child) .date-range {
+    margin-right: 1em;
+  }
+
+  .notes {
+    .note-metadata {
+      color: var(--typography-color-secondary);
+      filter: brightness(1.5); // muted font color, lighter than color-secondary
+    }
```
Comment (ayumi-oxd) [inline frontend/src/router/pages/ReviewChanges.scss:41]: Very cool, I didn't know this 😮

Comment (duncan-oxd) [reply frontend/src/router/pages/ReviewChanges.scss:41]: Ideally there would be a color in the design tokens for this, but the "primary" and "secondary" font colors look pretty much the same... Another thing I'll mention to Amanda when we go through things :P I think a custom shade of grey is probably fine to have for this.

SCSS has more color functions, but you'd need the colors to be SCSS variables to use them!
https://sass-lang.com/documentation/modules/color/

**Comment by duncan-oxd** [comment]
I'll add that "<- back to..." nav button on all of the pages in this branch, since it's the last ticket under CMS-422 👍

**Comment by duncan-oxd** [comment]
Just pushed a commit to add the "Back" links to the pages that needed it. I'll merge this and then go over the current state of everything with Amanda 👍

**Pull request merged**